docs/guide/contributing.md

```diff
@@ -0,0 +1,40 @@
+# Contributing to Echologator
+
+To contribute your changes to the documentation and the codebase, you would need to create a Pull Request.
+
```

Comment:
Can you quickly walk them through why they need to make a fork here. I imagine it would make a bit more sense if it went: How to Fork/Why Fork -----> How to PR/Why PR.

We don't want them to attempt to fork main xD

Reply:
Other than that it LGTM! :partying_face: